maintainer-nomination-docs/discord-message-template.md

```diff
@@ -31,12 +31,14 @@ Use this space to write a personal message in support of the nominee becoming ma
 ### Your Responsibility as a Voter:
 *(Please include this section as written, in its entirety)*
 
-Please reply to this thread with a clear vote within three days: yes, no, or abstain. Not voting within three days is equivalent to abstaining.
+Please vote in the poll on or before [3 business days in the future]. Not voting within this period is equivalent to abstaining.
```

Comment:
```suggestion
Please vote in the poll within 3 business days. Not voting within this period is equivalent to abstaining.
```

What do we think about this?

Reply:
I like this change personally, it's simple and efficient :thumbsup:

Reply:
✅